frontend/components/bc/BcTranslation.vue

```diff
@@ -0,0 +1,58 @@
+<script lang="ts" setup>
```

Comment:
Hmm we should somehow merge this with https://github.com/gobitfly/beaconchain/pull/642/files#diff-8285e5b745dca2f2f893bf9393584d96ede0d9df2b2aaf06a3f205247f500b22 that also planned for a url slot (but not tested) 

Your `link`slot would be the better choice, as I put mine more as a placeholder then a real implementation in. (as I only needed the bold highlighting for my content)

Reply:
So should this be done before or is it ok to do this later after I merged?

Reply:
Let's merge